en/config/actuators.md

```diff
@@ -0,0 +1,150 @@
+
+The screenshot below shows the geometry screen for a multicopter frame, which allows you to select the number of motors, their relative positions on the frame, and their expected spin directions (select "**Direction CCW**" for counter-clockwise motors).
+This particular frame also includes an image showing the motor positions, which dynamically updates as the motors settings are changed.
+
+![Geometry MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_geometry.png)
+
+A fixed wing airframe would instead display the parameters that define control surfaces, while a VTOL airframe would display both motors and control surfaces.
+
+
+### Conventions
+
+The following sections contain the conventions and explanations for configuring the geometry.
+
+#### Coordinate system
+
+The coordinate system is NED (in body frame), where the X axis points forward, the Y axis to the right and the Z axis down.
+Positions are relative to the center of gravity (in meters).
+
+#### Control Surfaces and Servo Direction
+
+Control surfaces use the following deflection direction convention:
+- horizontal (e.g. Aileron): up = positive deflection
+- vertical (e.g. Rudder): right = positive deflection
+- mixed (e.g. V-Tail): up = positive deflection
+
+![Plane Deflections](../../assets/config/actuators/plane_servo_convention.png)
+
+:::note
+If a servo does not move in the expected direction set in the geometry, you can reverse it using a checkbox on the Actuator Output.
+:::
+
+
+### Motor Tilt Servos
+
+Tilt servos are configured as follows:
+- The reference direction is upwards (negative Z direction).
+- Tilt direction: **Towards Front** means the servo tilts towards positive X direction, whereas **Towards Right** means towards positive Y direction.
+- Minimum and maximum tilt angles: specify the physical limits in degrees of the tilt at minimum control and maximum respectively.
+  An angle of 0° means to point upwards, then increases towards the tilt direction.
+  :::note
+  Negative angles are possible. For example tiltable multirotors have symmetrical limits and one could specify -30 as minimum and 30 degrees as maximum.
+  :::
+  :::note
+  If a motor/tilt points downwards and tilts towards the back it is logically equivalent to a motor pointing upwards and tilting towards the front.
+  :::
+- Control: depending on the airframe, tilt servos can be used to control torque on one or more axis (it's possible to only use a subset of the available tilts for a certain torque control):
+  - Yaw: the tilts are used to control yaw (generally desired).
+    If four or more motors are used, the motors can be used instead.
+  - Pitch: typically differential motor thrust is used to control pitch, but some airframes require pitch to be controlled by the tilt servos.
+    Bicopters are among those.
+- Tiltable motors are then assigned to one of the tilt servos.
+
+![Tilt Axis](../../assets/config/actuators/tilt_axis.png)
+
+
+### Bidirectional Motors
```

Comment:
@bkueng Not, I have updated this section from reversible to bidirectional motors. Why? Because we were also using reversible to mean a motor that can configure to go in either direction.

Do you think perhaps we could also change QGC to use the term "bidirectional"?

Reply:
@bkueng Did you see this? Are you otherwise good for this to merge?

Reply:
Yes bidirectional is fine.
- instead of 'required', can you use something like 'you have the option to...'? Even for rovers you don't need it, if you don't care about only going forwards.
- `the option will only be displayed if relevant for the frame type` -> you always have the option under 'advanced'

Reply:
> Yes bidirectional is fine.

It would be even better if you changed it in QGC :-).

I've added as (accepted) suggestions below. Note that when I pressed advanced checkbox in Sim last time I did not see the reversible option, which is why I assumed this is a feature of the airframe. Nevertheless I'll assume the sim is incorrect, not the intent.

Reply:
Already in the pipeline: https://github.com/PX4/PX4-Autopilot/pull/18990/commits/d23b5fdc0ea7f6afa47b9715e3d4723c523397b2 :-)
Hmm did you check on the geometry side?

![image](https://user-images.githubusercontent.com/281593/149079646-de667870-deb4-4452-989e-ee1136beb2c0.png)

Reply:
This is simulated iris from 23 December, with advanced checked.

![image](https://user-images.githubusercontent.com/5368500/149236900-69cb258d-81cd-4684-8a5f-048137afe833.png)

Reply:
Fixed in todays build:
![image](https://user-images.githubusercontent.com/5368500/149237648-ae945ca5-4e84-4150-a320-85e634a8f339.png)
So we can forget this